src/main/java/teammates/ui/template/CourseEditSectionRow.java

```diff
@@ -101,7 +101,15 @@ public ElementTag getToggleSessionLevelInSectionButton() {
         return toggleSessionLevelInSectionButton;
     }
     
-    private List<ElementTag> createPermissionInputGroup3ForSectionRow(int panelIndex) {
+    /**
+     * Creates checkboxes for Instructor's permissions/priviliges related to students' details,
+     *   comments for students (given by instructor or others) and automatically checks a single
+     *   checkbox if special privileges have been assigned to the section it corresponds to.
```

Comment:
The description for "students' details..." should be for `createCheckboxesForStudentPermissionsOfInstructors` instead.